backend/index.ts

```diff
@@ -35,3 +40,38 @@ const port = 8000;
 app.listen(8000, () => {
     console.log(`Server started: http://localhost:${port}/`);
 }); 
+
+app.get('/login/password',
+  function(req, res, next) {
+    res.render('login');
+  });
+
+app.post('/login/password',
+  passport.authenticate('local', { failureRedirect: '/login', failureMessage: true }),
+  function(req, res) {
+    console.log(req.body)
+    res.status(200).send("we logged in")
+  });
+
+const connection = mysql.createConnection({
+    host: process.env.DB_HOST,
+    user: process.env.DB_USER,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DB_NAME
+  });
+  
+passport.use(new LocalStrategy(function verify(username:any, password:any, cb:any) {
+    connection.query('SELECT * FROM users WHERE username = ?', [ username ], function(err:any, user:any) {
```

Comment:
So this is the way we would do it if we weren't using Prisma. Because we're using Prisma, we should be important Prisma models and using them instead.